### ISSUE TITLE
Add a A11y / accessibility tag in issue tracker + viewport issue

### ISSUE THREAD

**Issue by Facyla** [linagora/tmail-flutter#2864]
### Description
This ticket aims to foster the a11y topic into Twake apps, by providing a public category to help identifying and grouping relevant issues. This should also be done for all Twake apps, as some of the issues will be transverse to all the tools of that family. 

Also, here is a first a11y issue related to zooming and viewport settings, as an illustration of the use of that to-be category, from an auditing tool:
![image](https://github.com/linagora/tmail-flutter/assets/286492/f52a8c45-1a4e-4e16-8d95-0c05a77e2bea)

### Expected result
People should be able to zoom the page content.

### Current behavior
Unable to zoom.

### Preconditions (optional)
Because zoom is disabled in the app.

#### Reproduction Steps
Open Twake Mail on desktop in any browser. Use Ctrl + scroll or other standard means to zoom in/out => no effect.

### Acceptance criteria
Uwers should be able to adjust the content size to their needs. 

### Context
Desktop. OS / browser no relevant

**Comment by Facyla** [comment]
Here is the French referential on Accessibility that defines the requirements that the softwares operated by public tenants must respect https://accessibilite.numerique.gouv.fr/ 

Also here is the international referential on which the RGAA is partially based (many overlap but different criteria) : https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1

However, the target is not only to gain a good note, but to make sure that the applications can indeed be used by anyone, and in any situation, so an additional focus should be made throughout the overall user journey

**Comment by poupotte** [comment]
Linked to https://github.com/orgs/linagora/projects/3/views/1?pane=issue&itemId=131964395&issue=linagora%7Ctmail-flutter%7C4086
